arcgis-ios-sdk-samples/Scenes/Get Elevation at a point/GetElevationPointViewController.swift

```diff
@@ -0,0 +1,122 @@
+        let elevationSource = AGSArcGISTiledElevationSource(url: elevationURL!)
+        
+        // Add the elevation source to the surface.
+        surface.elevationSources.append(elevationSource)
+        scene.baseSurface = surface
+        
+        return scene
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Add the source code button item to the right of navigation bar.
+        (self.navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["GetElevationPointViewController", "ElevationViewController"]
+    }
+}
+
+// MARK: - AGSGeoViewTouchDelegate
+extension GetElevationPointViewController: AGSGeoViewTouchDelegate {
+    func geoView(_ geoView: AGSGeoView, didTapAtScreenPoint screenPoint: CGPoint, mapPoint: AGSPoint) {
+        if let relativeSurfacePoint = sceneView?.screen(toBaseSurface: screenPoint) {
+//            graphicsOverlay.graphics.removeAllObjects()
+            dismiss(animated: true)
+            
+            // Create the symbol at the tapped point.
+            let marker = AGSSimpleMarkerSceneSymbol(style: .sphere, color: .red, height: 100, width: 100, depth: 200, anchorPosition: .center)
+            let point = AGSPoint(x: relativeSurfacePoint.x, y: relativeSurfacePoint.y, spatialReference: .wgs84())
+            self.graphicsOverlay.graphics.add(AGSGraphic(geometry: point, symbol: marker))
+            
+            // Get the surface elevation at the surface point.
+            self.sceneView.scene?.baseSurface!.elevation(for: relativeSurfacePoint) { (results: Double, error: Error?) in
+                if let error = error {
+                    self.presentAlert(error: error)
+                } else {
+                    self.showPopover(elevation: results, popoverPoint: screenPoint)
+                }
+            }
+        }
+    }
+
+    private func showPopover(elevation: Double, popoverPoint: CGPoint) {
+        guard let controller = storyboard?.instantiateViewController(withIdentifier: "ElevationViewController") as? ElevationViewController else {
+                return
+        }
+        // Setup the controller to display as a popover.
+        controller.modalPresentationStyle = .popover
+        controller.loadViewIfNeeded()
+        controller.elevationLabel?.text? = "Elevation at tapped point: " + (String(elevation.rounded()) + String("m"))
+        controller.popoverPresentationController?.delegate = self
+        controller.preferredContentSize = CGSize(width: 280, height: 40)
+        controller.popoverPresentationController?.passthroughViews = [sceneView as Any, navigationController?.viewControllers as Any] as? [UIView]
+        controller.popoverPresentationController?.sourceRect = CGRect(origin: popoverPoint, size: .zero)
+        controller.popoverPresentationController?.sourceView = sceneView
+        present(controller, animated: true)
```

Comment:
I would suggest presenting the popover without animation so that the UI feels more responsive.

```suggestion
        present(controller, animated: false)
```